test/integration/TestHelper.cs

```diff
@@ -8,7 +8,7 @@ public class TestHelper
         public string ExampleCsprojFile { get; } = @"<Project Sdk=""Microsoft.NET.Sdk"">
   <PropertyGroup>
     <Description>An example csproj file.</Description>
-    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Is this the only functional change in this file? I appreciate you're probably using ReSharper and it's automatically re-arranged the members of this class. I'd prefer to do any sort of "cleanup" as part of a separate PR, though, and following a set of conventions that are explicitly defined for this project.